### ISSUE TITLE
WEBMAN MOD Latest makes my ps3 crash when playing games

### ISSUE THREAD

**Issue by goffrier** [aldostools/webMAN-MOD#557]
So i got my ps3 fat CECHK04 2 days ago, installed 4.84 dex and webman but had this issue that was persistent.
So it goes: PS3 Starts>I load black ops 2 disc via webman, get into rebug toolbox to spoof my dex to cex, R2+/_\ then i go ingame
but at the very last moment of loading multiplayer it crashes,3 beeps blinking red. That's all i know about this bug and i think i'm the first to experience it

**Comment by goffrier** [comment]
I removed webman 10 mins ago and it was all perfect except ofc i can't check temps anymore, no fan control, no ftp and all of that

**Comment by aldostools** [comment]
You can try installing an older build from the release page or directly from the commits

**Comment by goffrier** [comment]
okay thank you

**Comment by goffrier** [comment]
i'll try now :)

**Comment by goffrier** [comment]
thanks for those i'll definitely try those out

**Comment by goffrier** [comment]
so i reinstalled the cfw,still encountered issues with webman, multiman fine tho idk it's weird,only change i did is uninstall webman and change in rebug toolbox normal mode to rebug mode

**Comment by goffrier** [comment]
okay webman's at fault

**Comment by goffrier** [comment]
my test methodology is : 
1 - Boot PS3
2 - Boot Rebug Toolbox, Spoof to CEX
3 - Boot PSNPatch to disable CFW
4 - Boot MultiMAN and mount BO2 using the Spec Ops The Line disc
5 - Load BO2, Try to go online

I've had only failures with WEBMAN while without i was always successful

**Comment by goffrier** [comment]
i could test on 4.88 EVILNAT too if you want

**Comment by goffrier** [comment]
but it seems like it's just an issue specific to my console, i've searched online and there's nothing

**Comment by goffrier** [comment]
@aldostools it's related to fan control, i installed control fan utility and it did the same

**Comment by goffrier** [comment]
it even stayed after i uninstalled it so i had to install latest webman and remove it to clean whatever the fuck control fan utility installed

**Comment by aldostools** [comment]
I suggest that download the latest webMAN MOD and use SYSCON mode or use manual fan settings.

Maybe there is something wrong with your fans...

**Comment by goffrier** [comment]
okay

**Comment by goffrier** [comment]
i already use manual fan settings on webman tho

**Comment by goffrier** [comment]
i updated back to 4.88 evilnat, automatic fan control and it works?

**Comment by goffrier** [comment]
nvm it just happened now, automatic fan control just seem to delay the bug

**Comment by goffrier** [comment]
okay so syscon delays it further

**Comment by aldostools** [comment]
Probably manual setting would be your best option since the fan speed is constant.

**Comment by goffrier** [comment]
yeah but all the fan control settings just makes my ps3 crash with 3 beeps and red light blinking it seems, but i can't really leave it without webman or any sort of fan control since it will burn itself to death

**Comment by aldostools** [comment]
As this seems to be a hardware issue, I suggest you to ask in PSX-PLACE forums.
There you will receive help from various users with a lot of experience with PS3 hardware.

There is a thread that talk about how to change default fan settings in SYSCON.
https://www.psx-place.com/threads/modifying-syscon-fan-settings-better-than-webman.34626/

I haven't tried to do that, but it may worth to try.

**Comment by goffrier** [comment]
thank you

**Comment by goffrier** [comment]
okay so now when i uninstall webman mod my manual fan settings stays?

**Comment by NO-ob** [comment]
I've observed crashing on 4.88 evilnat with webman mod. Only seems to happen with modern warfare 2 so far I haven't tested many games but black ops and black ops 2 worked okay. It's not the iso or console because I tried both a us and and pal version of the game and experienced the same crashing it will cause black screen and the console needs to be force powered off. I confirmed this crashing on two different consoles too. I went out and bought a mw2 disc yesterday and that runs fine

**Comment by seedhelper** [comment]
having the same  issue on my fat ps3 with latest hen 3.11 using firmware 4.89

cpu 63    rsx 67  so its not a temps issue

i discovered that using webman to mount the game that i want causes de ps3 after some minutes to 3 beeps and red light blinking

if i want to use this console i have to run multiman and mount the game from there and voila   no issues

**Comment by seedhelper** [comment]
no you are not the only one mate.... thanks for your findings
